.github/ISSUE_TEMPLATE/3-introduce-application.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Adding an application
+about: Application Checklist
+title: Introduce <application>
+labels: "introduction, porter"
+assignees: "@nathankota"
+---
+
+# Summary
+
+- **Application Name**:
+- **Application Urls**
+  - Staging:
+  - Production:
+
+_A short summary of the situation._
+
+## Action items
+
+1. _Assign a person who should complete the task by replacing `name` with their github `@name`._
+1. _Check [x] the box when the task is completed and add the date of completion._
+1. _~Strike~ out all items that do not apply._
+
+- [ ] SOW or similar agreement (@nathankota, completed: `2022/00/00`)
+- [ ] Add to [gis.utah.gov](https://gis.utah.gov/developer/application) (name, completed: `2022/00/00`)
+- [ ] Create Archer security diagram (@nathankota, completed: `2022/00/00`)
+- [ ] Add to Application Porfolio Management ("Business Applications" in ServiceNow) (@nathankota, completed: `2022/00/00`)
+- [ ] Create maintenance contract (@nathankota, completed: `2022/00/00`)
+- [ ] Service Now task (@nathankota, completed: `2022/00/00`)
+- [ ] GitHub repo (name, completed: `2022/00/00`)
+  - Repo URL:
+- [ ] Primary client contact (@nathankota, completed: `2022/00/00`)
```

Comment:
```suggestion
- [ ] Create Archer security diagram (@nathankota, completed: `2022/00/00`)
  - Diagram URL: 
- [ ] Add to Application Portfolio Management ("Business Applications" in ServiceNow) (@nathankota, completed: `2022/00/00`)
  - Id: 
- [ ] Create maintenance contract (@nathankota, completed: `2022/00/00`)
- [ ] Service Now task (@nathankota, completed: `2022/00/00`)
  - Name: 
- [ ] GitHub repo (name, completed: `2022/00/00`)
  - Repo URL: 
- [ ] Primary client contact (@nathankota, completed: `2022/00/00`)
  - Name: 
```

Reply:
I applied this on my own since the suggestion became outdated.